lib/builder.js

```diff
@@ -122,7 +123,7 @@ function writeOutputFile(opts, output, basePath) {
   });
 }
 
-exports.inlineSourceMap = function(output) {
+var inlineSourceMaps = exports.inlineSourceMap = function(output) {
```

Comment:
Don't like that forward reference, but sure the exports do **pop** more this way.